global.json

```diff
@@ -2,8 +2,5 @@
   "projects": [
     "src",
     "test"
-  ],
-  "sdk": {
-    "version": "1.0.0-preview2-1-003180"
```

Comment:
It looks like every other repo has it, so fine, and we're going to migrate this repo to MSBuild soon anyway :).